js/projects.js

```diff
@@ -0,0 +1,217 @@
+            <li class="project-lang-item">css</li>
+            <li class="project-lang-item">bootstrap</li>
+            <li class="project-lang-item">ruby</li>
+          </ul>
+          <button id="4" class="button project-btn jsBtn">See Project</button>
+        </div>
+
+        <div id="project-5">
+          <h3 class="project-heading">Professional Art Printing Data</h3>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="project-lang">
+            <li class="project-lang-item">html</li>
+            <li class="project-lang-item">css</li>
+            <li class="project-lang-item">bootstrap</li>
+            <li class="project-lang-item">ruby</li>
+          </ul>
+          <button id="5" class="button project-btn jsBtn">See Project</button>
+        </div>
+
+        <div id="project-6">
+          <h3 class="project-heading">Professional Art Printing Data</h3>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="project-lang">
+            <li class="project-lang-item">html</li>
+            <li class="project-lang-item">css</li>
+            <li class="project-lang-item">bootstrap</li>
+            <li class="project-lang-item">ruby</li>
+          </ul>
+          <button id="6" class="button project-btn jsBtn">See Project</button>
+        </div>
+      </div>
+    </div>
+`;
+
+// CTREATE POP UP WINDOW
+const prjBtn = document.querySelectorAll('.jsBtn');
+prjBtn.forEach((button) => {
+  button.addEventListener('click', () => {
+    const section = document.createElement('section');
+    section.className = 'section-menu';
+    section.innerHTML = `
+    <div class="div-menu">
+      <h3 id="div-menu-h3">${projects[button.id].title}</h3>
+      <img id="menu-close-btn" src="./images/nav/close.png" />
+    </div>
+    <div class="div-menu">
+      <ul>
+        <li class="div-menu-li">qwerty</li>
+        <li class="div-menu-li">azerty</li>
+        <li class="div-menu-li">qwertz</li>
+      </ul>
+    </div>
+    <div id="div-section">
+      <div class="div-menu-img"><img id="div-section-img" src="${projects[button.id].image}" alt="" /></div>
+      <div id="div-section-menu">
+        <div class="div-menu"><p id="div-section-p">${projects[button.id].description}</p></div>
+        <div id="div-menu-last">
+          <a id="div-menu-live" href="${projects[button.id].live_version}"><button class="div-menu-button">See Live&nbsp;<div><img src="./images/social-media-header/live.png" alt="live icon" /></div></button></a>
+          <a id="div-menu-source" href="${projects[button.id].source_code}"><button class="div-menu-button">See Source&nbsp;<div><img src="./images/social-media-header/github.png" alt="" /></div></button></a>
+        </div>
+      </div>
```

Comment:
- [ ] Please as requested by the previous code reviewer, from the project requirements, you should stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=YHGkZAlalTNDuhZx-0) as much as possible. But as you can see you have some differences in the design. 

You can see the differences below your modal design and the Figma design does not match :-1: 


| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2022-11-26 03-55-08](https://user-images.githubusercontent.com/92303891/204071542-ff6543ef-386a-4c10-b04e-67bd90ead18b.png) | ![image](https://user-images.githubusercontent.com/92303891/204071599-10c6562e-b4bd-4359-b9f9-2e27f0d8b236.png) |